backend/test/modules/doubles-tournament/core/application/StartTournament.spec.ts

```diff
@@ -54,4 +54,27 @@ describe('Start tournament', () => {
     expect((commandResult as Failure).reason).toStrictEqual(new Error('Such tournament does not exists.'));
     expect(doublesTournament.lastPublishedEvent()).toStrictEqual(undefined);
   });
+
+  it('When command start tournament was called, but tournament under given id was already started, then tournament was not started and command failed', async () => {
+    //Given
+    const tournamentId = 'TournamentId';
+    const currentTime = new Date();
+    const team1Id = 'Team1Id';
+    const team2Id = 'Team2Id';
+    const entityIdGen = FromListIdGeneratorStub([team1Id, team2Id]);
+
+    const doublesTournament = testDoublesTournamentsModule(currentTime, entityIdGen);
+
+    //When
+    const startTournament = new StartTournament({
+      tournamentId: tournamentId,
+    });
+    await doublesTournament.executeCommand(startTournament);
+    const commandResult = await doublesTournament.executeCommand(startTournament);
+
+    //Then
+    expect(commandResult.isSuccess()).not.toBeTruthy();
+    expect((commandResult as Failure).reason).toStrictEqual(new Error('Such tournament has been already started'));
+    expect(doublesTournament.lastPublishedEvent()).toStrictEqual(undefined);
+  });
```

Comment:
ten nie przechodzi, ale to dlatego, że turniej o danym ID nie jest gotowy do wystartowania (czyli go nie ma jeszcze), dlatego mi się zatrzymuje na pierwszym if'ie w obiekcie domenowym...

Reply:
No to musisz go najpierw utworzyć w Given :)

Reply:
Sam sobie odpowiedziałeś na pytanie.

Reply:
Robiąc duże moduły potem testy wymagają coraz większych części Given - więcej musi się stać przed tym co testujemy.

Reply:
No i w sumie bardzo dobrze, że zatrzymuje się w pierwszym ifie, no bo przecież tak to właśnie zakodowałeś, że jeśli turniej nie istnieje to tak się stanie :)

Reply:
no tak może za szybko tutaj zapytałem - chodziło mi głównie o ten drugi test :-P

ale w sumie teraz podmieniłem w given doublesTournament na 
`    const doublesTournament = await createTournamentForTests(tournamentId, currentTime, team1Id, team2Id);
`
i robię przecież 2x execute to powinien teraz się wywalić na tym drugi execute... a skubany przechodzi

Reply:
dobra never mind .. działa .. oczywiście źle testowałem